Formula/libjwt.rb

```diff
@@ -0,0 +1,37 @@
+class Libjwt < Formula
+  desc "JSON Web Token C library"
+  homepage "https://github.com/benmcollins/libjwt"
+  url "https://github.com/benmcollins/libjwt/archive/v1.8.0.tar.gz"
+  sha256 "e97c15c14dbd49d5ba435cb26e06e6cdd6e32cf9f0be6b6e9a2d2d330923ea17"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "check" => :build
```

Comment:
Is this needed if we don't run `make check`?

Reply:
No, but we do run `make check` 🙂

Reply:
We generally only run `make check` for security software.

Reply:
👍